tests/modeltests/core_tests/tests/core.py

```diff
@@ -60,15 +60,27 @@ def test_deep(self):
         self.assertEqual(len(self.root_node.get_descendants()), 50 + 2)
 
     def test_validate_relationship_cls(self):
-        with self.assertRaises(ChildWasRejected):
+        try:
             self.widgy_site.validate_relationship(self.root_node.content, RawTextWidget)
+        except ChildWasRejected as e:
+            self.assertIn("Layout refuses to accept RawTextWidget.", str(e))
```

Comment:
Why can't we just do `self.assertIn(message, e.messages)` or whatever it should be? You get the idea...